sway/input/seat.c

```diff
@@ -1058,11 +1058,13 @@ void seat_set_raw_focus(struct sway_seat *seat, struct sway_node *node) {
 	}
 }
 
-void seat_set_focus(struct sway_seat *seat, struct sway_node *node) {
+static void _seat_set_focus(struct sway_seat *seat, struct sway_node *node, bool force) {
+	// If force is true, first unset fullscreen on overlapping node
```

Comment:
This comment probably belongs in the header file, distinguishing `seat_set_focus` from `seat_set_focus_force`.